device/mellanox/x86_64-mlnx_msn4800-r0/platform.json

```diff
@@ -0,0 +1,668 @@
+{
```

Comment:
@keboliu please check if this is aligned to the system definition. If we still don't have all the data, lets remove it and add when the hw is available (not the simulator)

Reply:
@keboliu kindly reminder